core/dop/autotest/testplan/autotest-plan.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package erda.core.dop.autotest.testplan;
+option go_package = "github.com/erda-project/erda-proto-go/core/dop/autotest/testplan/pb";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+service TestPlanService {
+  rpc UpdateTestPlanV2ByHook (TestPlanV2UpdateByHookRequest) returns (TestPlanV2UpdateByHookResponse)  {
+    option (google.api.http) = {
+      post: "/api/actions/autotest-plan-execute-callback",
+    };
+  }
+}
+
+// TestPlanV2UpdateByHooksRequest update testPlan by hook (post /api/actions/autotest-plan-execute-callback )
+message TestPlanV2UpdateByHookRequest{
```

Comment:
what is v2

Reply:
V2 is old name.
refactor to TestPlanUpdateByHookRequest